Projects/My_website/myWebsite/blog/urls.py

```diff
@@ -1,10 +1,15 @@
 from django.urls import path, include
 from rest_framework.routers import DefaultRouter
 from . import views
+from rest_framework import routers
 
 
+router = routers.DefaultRouter()
+router.register(r"", views.myBlog)
+
+router2 = routers.DefaultRouter()
+router2.register(r"", views.blog)
```

Comment:
@codal-uchoudhary 
why do we need 2 routers ?

Reply:
because we have two ModelViewSet class (myBlog , blog)

Reply:
okay, let me change this

Reply:
routers are updated